tributors/version.py

```diff
@@ -1,14 +1,14 @@
 """
 
-Copyright (C) 2020 Vanessa Sochat.
+Copyright (C) 2020-2022 Vanessa Sochat.
 
 This Source Code Form is subject to the terms of the
 Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
 with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 """
 
-__version__ = "0.0.19"
+__version__ = "0.0.2"
```

Comment:
???

Reply:
Pypi mustn't turn .20 into .2. why do you think it does?

Anyways, given that tributors is used in production, making it 0.1.0 makes  sense while at it

Reply:
all set! https://github.com/con/tributors/pull/62/commits/c92933ed4a63cf3b9a077a91af9b4e2d3969d9cf

Reply:
And you're right, but I had the wrong version. When I do `0.0.01` it changes to:

```
Creating tar archive
removing 'tributors-0.0.1' (and everything under it)
```
but `0.0.20` works so let's use that here instead - it's not really a major version bump for a bugfix.